### ISSUE TITLE
[Enhancement] Skinable Scrollbars

### ISSUE THREAD

**Issue by SunboX** [cdotyone/mochaui#94]
If I use the dark theme, the scrollbars stay system-grey ... would be cool if they were also skinable ;o)

**Comment by cdotyone** [comment]
agreed,  something I want, but will not make next version, unless someone else writes it.

**Comment by cleankod** [comment]
Very nice feature as long as those scrollbars can be operated with mouse scroll.
